### ISSUE TITLE
LB133 superseded by LB164?

### ISSUE THREAD

**Issue by abigailsnyder** [JGCRI/gcamdata#493]
The outputs from LB133 are used in LB164, but it seems as though these two chunks  could probably be combined in the future

**Comment by abigailsnyder** [comment]
*LB133.ag_Costs_USA_C_2005, LB164.ag_Costs_USA_C_2005,

**Comment by kvcalvin** [comment]
It looks like LB164 uses the outputs of LB133. And, L2052 uses something from L205. So, it may be easier to just do both???

**Comment by abigailsnyder** [comment]
I guess the main question is whether LB133 outputs are used anywhere else.
I believe, LB133 outputs = production costs for commodities.
LB164 outputs = production costs for commodities - purchased irrigation water.

**Comment by kvcalvin** [comment]
I think that is right. It was hard for me to tell whether LB133 outputs are used anywhere else that we plan to keep. I know they are used in L205 (which we don't plan to keep), but L205 outputs are used in L2052 (via L2051), which gets complicated. Maybe we wait on LB133 until we get to the L205 series?

**Comment by pkyle** [comment]
Just from a look, we can't really cut out the LB133 code; we could merge LB133 and LB164 into a single chunk, but it wouldn't really be cutting out any processing steps. In L205, the XML files generated are no longer going to be supported, and many of the CSVs generated are also obsolete. However, others are used later on, by L2051, L2052, L206, L2061, and L2071. Let me know if you want me to put together a list of which csv outputs are required for each of these chunks to produce. But I can guarantee you that LB133 is 100% required for LB164.

**Comment by bpbond** [comment]
>Let me know if you want me to put together a list of which csv outputs are required for each of these chunks to produce.

Remember we can do this automatically now!

**Comment by pkyle** [comment]
OK cool--my hypothesis is that excising the unnecessary parts of the existing data stream, and keeping the parts that are necessary to build the files called in the configuration file, won't be straightforward...prove me wrong!

**Comment by russellhz** [comment]
There does not appear to be anything obvious to do here. LB164 uses the outputs from LB133 and I don't see much repeated processing that could be avoided. To combine these we would just have to paste all of the code from LB133 at the start of LB164. I am going to close this for now.
